### PR TITLE
add highlight to table rows for features

### PR DESCRIPTION
**Pull request by ykhedher**
https://www.loom.com/share/658105486a074fc8a71a1f959bffcd53

### CODE REVIEW

**Comment by ykhedher** [comment]
[Add row highlight for features toggles list](https://trello.com/c/qoa2NFCb/578-add-row-highlight-for-features-toggles-list)

**Comment by FredrikOseberg** [comment]
>  [Add row highlight for features toggles list](https://trello.com/c/qoa2NFCb/578-add-row-highlight-for-features-toggles-list)

I like the highlighting. Can you navigate with the keyboard now the feature toggle screen?

**Comment by ykhedher** [comment]
Navigation with keyboard is fixed in this PR #479

**Comment by ivarconr** [comment]
Wow, the highlighting is pretty cool. 👍🏼

File: src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { makeStyles } from '@material-ui/core/styles';
 export const useStyles = makeStyles(theme => ({
     tableRow: {
         cursor: 'pointer',
+        '&:hover': {
+            backgroundColor: '#f5f5f5',
```
Comment (ivarconr) [inline src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts:7]: Can we use the color from the theme here, instead of hard-coding.

Comment (ykhedher) [reply src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts:7]: Yess. didn't pay attention to that sorry. I'll update it

**Comment by ykhedher** [comment]
https://www.loom.com/share/0c7dc21c637649d39d5c264fb77f62d5
Add highlight for:
- api list
- users list
- features list
- strategies list

File: src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts (hunk 1 above)
Comment (FredrikOseberg) [reply src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts]: You can use theme.palette.grey[100-1000]. So for example theme.palette.grey[500] - these are builtin shades of grey from material ui.

Comment (ykhedher) [reply src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts]: @FredrikOseberg I changed that in the new commit ✅

**Comment by FredrikOseberg** [comment]
I like the highlighting, but unsure if it makes sense in lists where there are no clickable actions. Any input on this?

**Comment by ykhedher** [comment]
@FredrikOseberg, Yes. for example the Apis list. they don't have clickable action. the goal is only to make it clear to the user that he's pressing the right button for that specific row. because sometimes you wanna make sure you are in the correct line. I think the highlight makes it super easy to make sure that you are in the right row. IDK if you understand what I meant.

**Review by ivarconr — APPROVED** [review]
This is very nice!

**Pull request merged**